coralproject/talk
# [CORL-625] Create link details in the moderation card details tabs

Adds copy-able link buttons to comment cards in the moderation queues in the details tabs.

## How to test

- Head to moderation queues
- Find a comment in the queues
- Click details on the comment card
- You should see two buttons to copy links
- Try them out, they link to single comment views for things
